src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/QueryStringValidation.feature.disabled

```diff
@@ -0,0 +1,194 @@
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?totalCount=false" 
+            """
+            """
+    Then it should respond with 200
+    And header "totalCount" is not present
+
+  @ignore
+  Scenario: Ensure that schools return the total count
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                    | gradeLevels                                                            | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]                                                    | [ "School" ]                         |
+            | 255901001    | Grand Bend High School               | [ "Tenth grade", "Ninth grade", "Eleventh grade", "Twelfth grade" ]    | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools" 
+            """
+            """
+    Then it should respond with 200
+    And header "totalCount" is not present
+
+
+  @ignore
+  Scenario: Ensure clients can GET information changing the casing of the query value to be all lowercase
+    Given this school exists:
+            | schoolId     | nameOfInstitution                    |                                                          | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=ut+austin+college+of+education+graduate" 
+            """
+            """
+    Then it should respond with 200
+    And "nameOfInstitution: UT Austin College of Education Under Graduate" 
+
+  @ignore
+  Scenario: Ensure clients can GET information changing the casing of the query value to be all uppercase
+    Given a school exists:
+            | schoolId     | nameOfInstitution                                    |                                                          | educationOrganizationCategories      |
+            | 6            | UT Austin College of Education Under Graduate        |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=UT+AUSTIN+COLLEGE+OF+EDUCATION+UNDER+GRADUATE" 
+            """
+            """
+    Then it should respond with 200
+    And "nameOfInstitution: UT Austin College of Education Under Graduate" 
+
+
+  @ignore
+  Scenario: Ensure clients can GET information filtering by totalCount
+    Given 8 schools exist:
+    When sending a GET request to "/ed-fi/schools?totalCount=true&limit=7" 
+    ##And "totalCount" 
+            """
+            """
+    Then getting less than the "totalCount"
+    And "totalCount: 7"
+
+  
+  @ignore
```

Comment:
The tests below belong in a different file, but we can make that change later in a future pull request for DMS-160.

Reply:
Sure, understood.